docs/fsharp/language-reference/operator-overloading.md

```diff
@@ -89,6 +89,8 @@ The following table shows the standard operators and their corresponding generat
 |`..`|`op_Range`|
 |`.. ..`|`op_RangeStep`|
 
+Note that the `not` operator in F# does not emit `op_Inequality` because it is not a symbolic operator.
```

Comment:
Can you say what `not` does emit? (I always ask that question whenever I read a doc that says what something *doesn't* do.)